docker-entrypoint.sh

```diff
@@ -2,8 +2,8 @@
 set -eu
 
 if [ "$(id -u)" == "0" ]; then
-  # Prepare default data directory
-  mkdir -p /data
+  # Prepare default data and config directories
+  chown -R symbolicator:symbolicator /etc/symbolicator
```

Comment:
I don't think this is necessary. `/etc/symbolicator` isn't meant to be writable (e.g., we currently mount it in read-only mode), so it can potentially break things.

Reply:
WFM, as long as the files are readable by symbolicator user by default (and I think that is the case right now).